src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -4,6 +4,10 @@ setlocal
 set PROJECT_DIR=%~1
 set DOTNET_CMD=%~2
 set TOOLRUNTIME_DIR=%~3
+set PACKAGES_DIR=%4
+if [%PACKAGES_DIR%] == [] set PROJECT_DIR=%PROJECT_DIR%\packages\
```

Comment:
Sounds good. I just pushed a rebased commit to fix the merge conflicts and includded the fix from your feedback.